src/mobot_client/admin.py

```diff
@@ -62,8 +61,8 @@ class BonusCoinAdmin(admin.ModelAdmin):
     readonly_fields = ('number_claimed', 'number_remaining',)
```

Comment:
Now that number_claimed is a database field and not calculated, for testing is sure is helpful to have it be writable from the admin interface.